Package.swift

```diff
@@ -35,9 +35,12 @@ let package = Package(
         .testTarget(
             name: "MatchingEngineTests",
             dependencies: ["_MatchingEngine"]),
+        .target(
+            name: "_CUnicode",
+            dependencies: []),
         .target(
             name: "_StringProcessing",
-            dependencies: ["_MatchingEngine"],
+            dependencies: ["_MatchingEngine", "_CUnicode"],
```

Comment:
Hmm I don't think we can add arbitrary dependencies for `_MatchingEngine` or `_StringProcessing` until we 1) set up cmake on the compiler side to produce the _CUnicode module and 2) coordinate with @edymtt for internal builds.

Reply:
Is there any way to avoid `_CUnicode`?

Reply:
Maybe you can build the C files as part of _StringProcessing via CMake? But I’m not sure how you can do that using SwiftPM.

Reply:
I don't see any reason why the two build systems _have_ to work the same way. If we need to include these directly in the CMake build for now, let's do that.